Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -647,6 +643,29 @@ function vtkResliceCursorWidget(publicAPI, model) {
       );
     }
   );
+
+  publicAPI.getPlaneExtremities = (viewType) => {
+    const dirProj = publicAPI.getWidgetState().getPlanes()[viewType].normal;
+    const p1 = vtkMath.multiplyAccumulate(
+      publicAPI.getWidgetState().getCenter(),
+      dirProj,
+      -10000,
+      []
+    );
+    const p2 = vtkMath.multiplyAccumulate(
+      publicAPI.getWidgetState().getCenter(),
+      dirProj,
+      10000,
```

Comment:
These seem like "sufficiently large" constants for most realistic data, but still has edge case behavior for images with very large bounds. p1 and p2 could end up inside the bounds, which means instersectionPoints is empty (presuming intersectWithLine does segment-plane intersection).

Reply:
should now be fixed